oat-sa/tao-test-runner-qti-fe
# Fix/BSA-219/timers don't pause in server mode

Related to: https://oat-sa.atlassian.net/browse/BSA-219

For timer mode `server` don't pause timers.

How to test:

1. check that timer mode is  `server` `config/taoQtiTest/testRunner.conf.php` `'timer' => array('target' => 'server'),`
2. create test with Navigation Warnings  `Display Next Section Warning` and `Display End Test Warning`
3. Set a time limit
4. create delivery
5. run test
6. check that timer doesn't pause while popup is displayed
